src/content/content.ts

```diff
@@ -353,6 +353,60 @@ export const nameYourTemplateContent = {
   },
 };
 
+export const channelGuidanceContent = {
+  [TemplateType.NHS_APP]: {
+    heading: 'More about NHS App messages',
+    guidanceLinks: [
+      {
+        text: 'NHS App messages (opens in a new tab)',
+        link: 'https://notify.nhs.uk/features/nhs-app-messages',
```

Comment:
Don't include the domain, the user should be kept in the current domain, all requests go through the web gateway.  Follow the pattern of the accessibility link.

Reply:
updated